use-timescale/security/strict-ssl.md

```diff
@@ -111,11 +111,13 @@ To set up a stricter SSL connection:
 
 ## Verify the certificate type used by your database
 
-To check whether the certificate has been replaced yet, connect to your database
-instance and inspect the returned certificate:
+To check whether the certificate has been replaced yet, connect to your
+database instance and inspect the returned certificate. We are using two
+certificate providers - Google and ZeroSSL, that's why chances are you can have
```

Comment:
certificate providers—Google and ZeroSSL. That's why chances are you can have
[...]